packages/epics/src/execute/execute.ts

```
@@ -372,6 +376,8 @@ export function sendExecuteRequestEpic(
              * We cannot execute cells with no content, so
              * we through an error action if this is the case.
              */
+            // RID:JK this is also causing distracting notifications
+            // in normal use. Should be handled similar to markdown cells
```

I think we could remove these comments if the PR resolves this problem. 

Depending on your final solution here, I might edit the docstring on line ~348 to reflect the new behavior. Something like "attempts to execute non-code cells will fail silently"